src/app/components/SocialLogin/SocialLogin.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import FacebookIcon from '../Icons/FacebookIcon';
+import GoogleIcon from '../Icons/GoogleIcon';
+import SocialLogin from './SocialLogin';
+
+export default {
+  title: 'Component/SocialLogin',
+  component: SocialLogin,
+};
+
+export const SocialLogins = (): JSX.Element => (
+  <SocialLogin
+    title="Social Logins"
+    icon1={<FacebookIcon />}
+    icon2={<GoogleIcon />}
```

Comment:
I would love to see a list of social icons here to allow adding more services without changing the component.
wdyt about:
```suggestion
    icons={[{
      href: "https://facebook.com/photoplay",
      Icon: FacebookIcon
    }, {
      href: "https://google.com/photoplay",
      Icon: GoogleIcon 
    }]}
```

or:
```suggestion
    icons={[{
      href: "https://facebook.com/photoplay",
      icon: <FacebookIcon />
    }, {
      href: "https://google.com/photoplay",
      icon: <GoogleIcon />
    }]}
```